src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/dml/07 PeriodEnd.DataLock.Populate.Reference.ApprenticeshipPriceEpisode_Period.dml.sql

```diff
@@ -9,24 +9,30 @@ INSERT INTO [Reference].[ApprenticeshipPriceEpisode_Period] (
 	[PriceEpisodeFirstEmp1618Pay],
 	[PriceEpisodeSecondEmp1618Pay]
 	)
-	SELECT
-	ape.[Ukprn] ,
-	ape.[LearnRefNumber] ,
-	ape.[PriceEpisodeIdentifier] ,
-	[Period] ,
-	p.[PriceEpisodeFirstEmp1618Pay],
-	p.[PriceEpisodeSecondEmp1618Pay]
-
-	
-	FROM ${ILR_Deds.FQ}.[Rulebase].[AEC_ApprenticeshipPriceEpisode] ape
-	JOIN ${ILR_Deds.FQ}.[Rulebase].[AEC_ApprenticeshipPriceEpisode_Period] p
-		On p.Ukprn = ape.Ukprn
-		And p.LearnRefNumber = ape.LearnRefNumber 
-		And p.PriceEpisodeIdentifier = ape.PriceEpisodeIdentifier
-
-    WHERE ape.PriceEpisodeContractType = 'Levy Contract'
-	And ape.UKPRN IN (SELECT DISTINCT [Ukprn] FROM [Reference].[Providers])
-	And (IsNull(p.PriceEpisodeFirstEmp1618Pay,0) <> 0 OR IsNull(p.PriceEpisodeSecondEmp1618Pay,0) <> 0)
-	
-	
-	 
+SELECT 
+	[Ukprn],
+    [LearnRefNumber],
+    [PriceEpisodeIdentifier],
+    [Period],
+    [PriceEpisodeFirstEmp1618Pay],
+    [PriceEpisodeSecondEmp1618Pay]
+FROM 
+	OPENQUERY(${DS_SILR1718_Collection.servername}, '
+	SELECT 
+		ape.[Ukprn],
+		ape.[LearnRefNumber],
+		ape.[PriceEpisodeIdentifier],
+		[Period],
+		p.[PriceEpisodeFirstEmp1618Pay],
+		p.[PriceEpisodeSecondEmp1618Pay],
+		ape.PriceEpisodeContractType
+	FROM
+		${DS_SILR1718_Collection.databasename}.[Rulebase].[AEC_ApprenticeshipPriceEpisode] ape
+		INNER JOIN ${DS_SILR1718_Collection.databasename}.[Rulebase].[AEC_ApprenticeshipPriceEpisode_Period] p ON p.Ukprn = ape.Ukprn
+			AND p.LearnRefNumber = ape.LearnRefNumber
+			AND p.PriceEpisodeIdentifier = ape.PriceEpisodeIdentifier') as ape
+WHERE 
+	PriceEpisodeContractType = 'Levy Contract'
```

Comment:
Again, would it be better to move some of the WHERE predicates into the OPENQUERY?